.travis.yml

```diff
@@ -14,8 +14,9 @@ services:
 
 env:
   matrix:
-  - NODE_VERSION=8
-  - NODE_VERSION=9
+  - NODE_VERSION=8 TEST_SUITE=normal
+  - NODE_VERSION=9 TEST_SUITE=normal
+  - NODE_VERSION="lts/*" TEST_SUITE=performance
```

Comment:
does that mean it runs on all lts builds? Surely just running it on 8 or 9 would be good enough?

Reply:
no, it means it runs on the latest lts build

Reply:
e.g. 
```
$ nvm install lts/*
v8.10.0 is already installed.
Now using node v8.10.0 (npm v5.6.0)
$
```